workflow/scripts/build_scenarios.py

```diff
@@ -45,6 +45,30 @@ def get_shares(df, planning_horizons):
 
     return transport_share, naval_share
 
+def get_transport_growth(df, planning_horizons):
+    # Aviation growth factor - using REMIND-EU v1.1 since DEMO v1 does not include bunkers
+    aviation_model = "REMIND-EU v1.1"
+    aviation = df.loc[aviation_model,"Final Energy|Bunkers|Aviation"]
+    aviation = aviation[planning_horizons]
+    aviation.reset_index(inplace=True)
+    aviation.drop(columns="unit", inplace=True)
```

Comment:
I would prefer if we don't just drop the unit, but include it when accessing `.loc[:, :, unit]`. That way the units are explicitly stated in the code (and we save the slightly obscur `.drop(column="unit")`)

Reply:
I kept the information within the function but it does not make sense to pass it along since they are dimensionless factors